README.md

```diff
@@ -43,7 +43,7 @@ The base URL your site is located at. Currently used in combination with other s
 
 `URI_ALLOW_LIST` - `string`
 
-A comma separated list of URIs (e.g. "https://supabase.io/welcome,io.supabase.gotruedemo://logincallback") which are permitted as valid `redirect_to` destinations, in addition to SITE_URL. Defaults to []. Supports wildcard matching trough globbing. e.g. add `*.mydomain.com/welcome` -> `x.mydomain.com/welcome` and `y.mydomain.com/welcome` would be accepted.
+A comma separated list of URIs (e.g. `"https://foo.example.com,https://*.foo.example.com,https://bar.example.com"`) which are permitted as valid `redirect_to` destinations. Defaults to []. Supports wildcard matching through globbing. e.g. `https://*.foo.example.com` will allow `https://a.foo.example.com` and `https://b.foo.example.com` to be accepted. For more common glob patterns, check out the [following link](https://pkg.go.dev/github.com/gobwas/glob#Compile).
```

Comment:
Looks great! Can we highlight support for globbing with subdomains (e.g. https://foo.example.com/* like in [this test](https://github.com/supabase/gotrue/blob/master/api/verify_test.go#L492)) think we had questions on. that in the past.